lib/launcher/revision.go

```diff
@@ -3,4 +3,4 @@
 package launcher
 
 // DefaultRevision for chromium
-const DefaultRevision = 993261
+const DefaultRevision = 983172
```

Comment:
How did you find out this revision works?

Reply:
Revision is based on the [`BUILD_NUMBER`](https://github.com/microsoft/playwright/blob/main/browser_patches/chromium/BUILD_NUMBER) in Playwright's repo and the zipName is in [`EXPECTED_BUILDS`](https://github.com/microsoft/playwright/blob/main/browser_patches/chromium/EXPECTED_BUILDS), but after chromium 100 the number is not the same as the chromium revision in Google snapshots and NPM mirror, or it's better to follow Playwright's build version in another way?